reagurk/public/index.html

```diff
@@ -4,11 +4,20 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Gurkult - An Open Source Community"
-    />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://gurkult.com" />
+    <meta property="og:title" content="Gurkult" />
+    <meta property="og:site_name" content="Gurkult" />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:description" content="The Gurkult is an open source community website."/>
+    <meta property="og:image" content="%PUBLIC_URL%/logo.png" />
+    <meta propert="og:image:alt" content="Gurkult Logo" />
+    <meta name="twitter:card" content="summary_small_image" />
```

Comment:
```suggestion
    <meta name="twitter:card" content="summary_large_image" />
```
This is the large card that is typically used.